monkestation/code/modules/virology/immune_systems/_immune_system.dm

```diff
@@ -19,7 +19,8 @@
 		ANTIGEN_Z	= 0,
 		)
 
-/datum/immune_system/Destroy()
+/datum/immune_system/Destroy(force)
```

Comment:
DOUBLE CALL, LUCCCCCCYYYYY, DOUBLE CALLLLLLLLLLLL(waltuh.png)

Reply:
thanks for catching that!